src/axom/slic/tests/slic_benchmark_asserts.cpp

```diff
@@ -3,41 +3,43 @@
 //
 // SPDX-License-Identifier: (BSD-3-Clause)
 
-#include "benchmark/benchmark_api.h"
+#include "benchmark/benchmark.h"
 #include "axom/slic/interface/slic.hpp"
 #include "axom/slic/core/SimpleLogger.hpp"
 
 /*!
  * \file
  *
- * A series of tests of SLIC logging macros (SLIC_WARNING to avoid killing the
- * program) in the TPL benchmark framework.
+ * A series of tests of SLIC logging macros (using SLIC_WARNING to avoid 
+ * killing the program) in the TPL benchmark framework.
  *
  * These tests were written to see why SLIC macros was not being logged
  * when called from a benchmark fixture's destructor.
  * It turns out that the fixtures are destructed after main exits,
  * so SLIC is already finalized.
- *
  */
 
 namespace
 {
-static const bool IS_VALID = true;
-static const bool SHOULD_PRINT = false;
+static constexpr bool IS_VALID = true;
+static constexpr bool SHOULD_PRINT = false;
 
 void printMsg(std::string const& str, bool override = false)
 {
-  if(SHOULD_PRINT || override) std::cout << str << std::endl;
+  if(SHOULD_PRINT || override)
```

Comment:
"override" is just a keyword in some contexts but since C++11 I think twice about using it for identifiers.

Reply:
Good call -- I think that was written before we moved to C++11. I'll rename it to avoid possible confusion

Reply:
done